src/styles/styles.css

```diff
@@ -1,4 +1,128 @@
 /* Global styles for project */
 body {
-  background-color: #F5F5F5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('../assets/background-image.jpeg');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-color: grey;
+}
+
+#app {
+  width: 90%;
+  color: #ffb400;
+  font-family: Titillium Web, sans-serif;
+}
+
+h1 {
+  margin-bottom: 40px;
+  background-color: grey;
+}
+
+@keyframes stripe-slide {
+  0% {
+    background-position: 0% 0;
+  }
+  100% {
+    background-position: 100% 0;
+  }
+}
+
+.get-trains-btn {
+  height: 35px;
+  padding-bottom: 15px;
+  border-radius: 6px;
+  background-color: #ffb400;
+  border-color: black;
+  text-transform: uppercase;
+  font-family: Titillium Web, sans-serif;
+  font-weight: bold;
+  font-size: 16px;
+  cursor: pointer;
+  outline: none;
+}
+
+.btn--stripe {
+  overflow: hidden;
+  position: relative;
+}
+
+.btn--stripe:after {
+  content: '';
+  display: block;
+  height: 7px;
+  width: 100%;
+  background-image: repeating-linear-gradient(45deg, #666, #666 1px, transparent 2px, transparent 5px);
```

Comment:
Please, use fallback background property for gradients: https://css-tricks.com/almanac/properties/b/background-image/#article-header-id-2

Reply:
Good point, but not suitable for this case. This gradient looks like zebra, so if I set fallback color it visible under gradient. But I will note it to self.

Reply:
It's just fallback. Any background is better than none.

If the browser does not support gradients, the area will still have a background that you set.